renderdoc/driver/vulkan/vk_core.cpp

```diff
@@ -1450,11 +1447,10 @@ bool WrappedVulkan::EndFrameCapture(void *dev, void *wnd)
       {
         for(uint32_t x = 0; x < thwidth; x++)
         {
-          float xf = float(x) / float(thwidth);
-          float yf = float(y) / float(thheight);
+          uint32_t xSource = x * imInfo.extent.width / thwidth;
+          uint32_t ySource = y * imInfo.extent.height / thheight;
```

Comment:
This fix is good but should be propagated to the other APIs as well. See other comment about the typed thumbnails in general, it's up to you if you want to refactor where this resample happens at the same time or not.

Reply:
I'd rather keep the thumbnail and resampling split as CLs in principle, they are really not related i.e. I can apply resampling fix separately from the thumbnail-format change. 
Just to clarify, what's your resolution on this CL?.. Do you want me to change this CL to apply the the fix across other APIs or just have follow up CLs?..

Reply:
Like I say it's up to you how you want to handle it. If you want the fix in now then I'd like it to be applied to all the other APIs, I just thought that if you feel like refactoring this code into a single place while looking at the typed thumbnails, you might want to save yourself the redundant work, but I don't have any preference either way.